src/IdToUuidMigration.php

```diff
@@ -37,16 +37,16 @@ public function up(Schema $schema):void
     {
     }
 
-    public function migrate(string $tableName)
+    public function migrate(string $tableName, string $uuidColumnName = '__uuid__')
```

Comment:
To avoid BC, you should maybe default it to 'uuid'. Defaulting it to '__uuid__' will  break any apps that were using this

Reply:
It's only a temporary name, so it should be fine, using `__uuid__` will reduce risk of any conflicts. LGTM